Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM apache/airflow:2.9.3-python3.12
+FROM apache/airflow:2.10.4-python3.12
```

Comment:
Updating to the latest version of Airflow. I've run the harvest DAG locally but not in our "production" environment.